_posts/2018-08-06-better-bash-in-5-minutes.markdown

```diff
@@ -1,11 +1,7 @@
 ---
 layout: post
 title:  "Better Bash Scripting in 5 Minutes or Less"
-<<<<<<< HEAD
-date:   2018-08-06
```

Comment:
don't you want this date? Per your file name? Or is the filename wrong

Reply:
date for Monday (next) please

Reply:
which it is, right?

Reply:
what

Reply:
happens

Reply:
if

Reply:
I